### DISCUSSION TITLE
looking  for desktops

### DISCUSSION THREAD

**Discussion by BOWIE1208** [rocksdanister/lively#1971] [General]
[https://www.rocksdanister.com/lively/](url)
The Great Wave.....thanks

**Comment by rocksdanister** [comment]
It should be on here:
https://www.reddit.com/r/LivelyWallpaper/comments/kfvc2o/how_to_download_wallpapers_from_lively_deviantart/

**Reply by BOWIE1208** [reply]
Much appreciated
